swagger.json

```diff
@@ -5850,6 +6001,72 @@
             "items": {
                 "$ref": "#/definitions/DocumentType"
             }
+        },
+        "User": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "description": "The id of the User record.",
+                    "example": 123
+                },
+                "name" : {
+                    "type": "string",
+                    "description": "The first and last name of the user.",
+                    "example": "Jane Doe"
+                },
+                "email": {
+                    "type": "string",
+                    "description": "The email address of this user.",
+                    "example": "user@company.com"
+                },
+                "organizationRole" : {
+                    "type": "string",
+                    "description": "The name of the role the user is assigned to for the organization. This will be blank for users that only have access to specific tags.",
+                    "example": "Read-Only Admin"
```

Comment:
We should expose this as an enum field: https://swagger.io/docs/specification/2-0/enums/